lib/python/satos_payload_sdk/app_framework.py

```diff
@@ -221,7 +222,13 @@ def stage_file_download(self, loc):
         return resp
 
     def payload_power_control(self):
-        raise NotImplementedError()
+        # In case if True-twin mode, return success
+        if api_common.g_KEEPALIVE_ENABLE == '1':
```

Comment:
isn't there a flag called "remote_payload" or  "TrueTwin_mode" or something like that ?

Reply:
Currently it is not there. I am filing Jira for the same.